test/components/routers/test_transformers_text_router.py

```diff
@@ -80,6 +80,27 @@ def test_from_dict(self, mock_auto_config_from_pretrained, monkeypatch):
             "token": None,
         }
 
+    @patch("haystack.components.routers.transformers_text_router.AutoConfig.from_pretrained")
+    def test_from_dict_no_default_parameters(self, mock_auto_config_from_pretrained, monkeypatch):
+        mock_auto_config_from_pretrained.return_value = MagicMock(label2id={"en": 0, "de": 1})
+        monkeypatch.delenv("HF_API_TOKEN", raising=False)
+        data = {
+            "type": "haystack.components.routers.transformers_text_router.TransformersTextRouter",
+            "init_parameters": {"model": "papluca/xlm-roberta-base-language-detection"},
```

Comment:
Wondering is there a reason, a default model is not set for `transformers_text_router`?

Reply:
Yeah this was a choice because Text Classification models vary widely about what labels they can classify. And there is no clear "best" model since it will depend on what classification task the end user wants. I was afraid setting a default model would imply to users that this component can only classify the labels associated with the chosen default model. This hopefully forces them to find the model that actually suites their needs.